<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SAIF" name="bdd.runners.RunnerClass" tests="3" failures="1" timestamp="6 Aug 2020 02:49:19 GMT" time="79.029" errors="0">
  <testcase name="feature" time="29.626" classname="bdd.runners.RunnerClass"/>
  <testcase name="feature" time="20.242" classname="bdd.runners.RunnerClass">
    <failure type="cucumber.runtime.CucumberException" message="junit.framework.ComparisonFailure: expected:&lt;$[0.0]0&gt; but was:&lt;$[1.8]0&gt;">
      <![CDATA[cucumber.runtime.CucumberException: junit.framework.ComparisonFailure: expected:<$[0.0]0> but was:<$[1.8]0>
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)
Caused by: junit.framework.ComparisonFailure: expected:<$[0.0]0> but was:<$[1.8]0>
	at junit.framework.Assert.assertEquals(Assert.java:85)
	at junit.framework.Assert.assertEquals(Assert.java:91)
	at bdd.usps.PageActions.UspsCalculatePriceActions.verifyTotalPrice(UspsCalculatePriceActions.java:49)
	at bdd.usps.stepDef.UspsCalculatePriceSteps.varify_the_total_price_is_correct(UspsCalculatePriceSteps.java:33)
	at ?.Then Varify the total price is correct(03.UspsCalculateAPrice.feature:9)
]]>
    </failure>
  </testcase> <!-- feature -->
  <testcase name="feature" time="29.161" classname="bdd.runners.RunnerClass"/>
</testsuite> <!-- bdd.runners.RunnerClass -->
